### DISCUSSION TITLE
Power - offtemp should be controllable

### DISCUSSION THREAD

**Discussion by paulbusse** [dominikamann/oekofen-pellematic-compact#87] [Ideas]
Hi,

First of all thanks for the integration. Much better than the one I created (using MQTT).

In principle, I think you should make  all fields not starting with 'L_' controllable.

The one that I'm missing is power - offtemp.
I use this as follow: 
* offtemp = 60° => first send electricity to the buffer
* When the buffer is at 60°, all electricity goes to the car
* When the car is full: set offtemp to 90°: remaining electricity goes to the buffer

As far as I can see in `const.py` power - offtemp is not controllable

Thanks 

Paul

**Comment by dominikamann** [comment]
Hi @paulbusse,

please share the JSON of the missing values with the additional infos (/all?).

Thanks
